automated/linux/iperf/iperf-server.yaml

```diff
@@ -0,0 +1,40 @@
+metadata:
+    name: iperf
+    format: "Lava-Test-Shell Test Definition 1.0"
+    description: "iperf is a tool for active measurements of the maximum
+                  achievable bandwidth on IP networks."
+    maintainer:
+        - milosz.wasilewski@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+        - openembedded
+    scope:
+        - performance
+    environment:
+        - lava-test-shell
+    devices:
+        - hi6220-hikey
+        - apq8016-sbc
+        - mustang
+        - moonshot
+        - thunderX
+        - d03
+        - d05
+        - soca9
+        - rzn1d
+
+params:
+    SKIP_INSTALL: "false"
+    SERVER_ETHERNET_DEVICE: "eth0"
+
+run:
+    steps:
+        - cd ./automated/linux/iperf/
+        - ipaddr=$(lava-echo-ipv4 ${SERVER_ETHERNET_DEVICE} | tr -d '\0')
```

Comment:
the error is printed to stderr so it's not harmful. I would add one more line after this:
```
if [ -z "${ipaddr}" ]; then echo lava-test-raise "${SERVER_ETHERNET_DEVICE} not found"; fi
```
This will abort the test in case the interface doesn't exist. IHMO this is right thing to do.

Reply:
Ack.

Reply:
OK, done in the latest push.